diff --git a/include/downloader.h b/include/downloader.h
index b60e481..112c2a0 100644
--- a/include/downloader.h
+++ b/include/downloader.h
@@ -25,6 +25,7 @@
 #include <map>
 #include <pthread.h>
 #include <memory>
+#include <stdexcept>
 
 namespace kiwix
 {
diff --git a/src/tools/regexTools.cpp b/src/tools/regexTools.cpp
index 9144040..1b70198 100644
--- a/src/tools/regexTools.cpp
+++ b/src/tools/regexTools.cpp
@@ -25,6 +25,7 @@
 
 #include <memory>
 #include <map>
+#include <stdexcept>
 #include <pthread.h>
 
 std::map<std::string, std::shared_ptr<icu::RegexPattern>> regexCache;
