From 7124397da4f5743ed0ebdb59b54d79d551dfccbf Mon Sep 17 00:00:00 2001
From: Vitaly Zaitsev <vitaly@easycoding.org>
Date: Wed, 10 Apr 2019 15:46:08 +0200
Subject: [PATCH] Removed -Werror build flag.

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 24493b6..4152625 100644
--- a/meson.build
+++ b/meson.build
@@ -1,7 +1,7 @@
 project('kiwix-tools', 'cpp',
   version : '1.1.0',
   license : 'GPL',
-  default_options: ['c_std=c11', 'cpp_std=c++11', 'werror=true'])
+  default_options: ['c_std=c11', 'cpp_std=c++11'])
 
 compiler = meson.get_compiler('cpp')
 
-- 
2.20.1

