From 745aebff7e72ac707aed6dcc502fd47d9dceac7c Mon Sep 17 00:00:00 2001
From: Vitaly Zaitsev <vitaly@easycoding.org>
Date: Wed, 10 Apr 2019 15:21:36 +0200
Subject: [PATCH 1/2] Removed Rpath build.

---
 src/meson.build | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/meson.build b/src/meson.build
index 17fdff9..98167c8 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -44,5 +44,4 @@ kiwixlib = library('kiwix',
                    dependencies : all_deps,
                    version: meson.project_version(),
                    install: true,
-                   install_dir: install_dir,
-                   install_rpath: '$ORIGIN')
+                   install_dir: install_dir)
-- 
2.20.1

